src/main/scala/com/mesosphere/cosmos/TestProperty.scala

```diff
@@ -1,9 +1,8 @@
 package com.mesosphere.cosmos
 
 /**
- * Specifies how to wire a system property into both the server under test and the test client.
+ * Specifies how to wire a system property for the server under test.
  *
- * @param clientName - the name of the test client that makes use of the property value.
```

Comment:
You removed the code that uses `clientName` -- can we get rid of the field? And at that point, maybe this entire class isn't needed anymore?